### ISSUE TITLE
RoleFillList method throws exception when deserializing the response

### ISSUE THREAD

**Issue by dwcai** [Parature/ParatureSDK#11]
When trying to retrieve the list of available customer roles, I am getting the following error message. 
```
System.InvalidOperationException: There is an error in XML document (0, 0).
  Stack Trace: 
   at System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader, String encodingStyle, XmlDeserializationEvents events)
   at System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader)
   at ParatureSDK.XmlToObjectParser.ParaEntityParser.EntityFill[T](XmlDocument xmlDoc)
   at ParatureSDK.XmlToObjectParser.ParaEntityParser.FillList[TEnt](XmlDocument xmlDoc)
   at ParatureSDK.ApiHandler.Entities.Role.RoleFillList(ParaCredentials paraCredentials, RoleQuery query, ParatureModule Module)
   at ParatureSDK.ApiHandler.Entities.Role.RolesGetList(ParaCredentials ParaCredentials, RoleQuery Query, ParatureModule Module) 

System.InvalidOperationException: <CustomerRole xmlns=''> was not expected.
  Stack Trace: 
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderRole.Read3_Role() 

```

The problem is, the returned XML nodes have a name of CustomerRole while the serializer is expecting Role only.

**Comment by brtubb** [comment]
Thanks for reporting this! It isn't fixed yet sadly. I'll take a look at it soon. Thank you again for the detailed debugging information.

**Comment by brtubb** [comment]
Happy to report this is fixed in 2.0.4 on NuGet and in master. The API namespaces have been modified a bit to accommodate. Specifically, you can handle Status/Role/View retrieval for all entities that support it using the "GetList" or "GetDetails" methods. Looks like:

```
            var ticketStatus = ApiHandler.Ticket.Status.GetList(creds);
            var ticketView = ApiHandler.Ticket.View.GetList(creds);
            var custRole = ApiHandler.Customer.Role.GetList(creds);
            var custStatus = ApiHandler.Customer.Status.GetList(creds);
            var custView = ApiHandler.Customer.View.GetList(creds);
            var csrRole = ApiHandler.Csr.Role.GetList(creds);
            var csrStatus = ApiHandler.Csr.Status.GetList(creds);
            var accountView = ApiHandler.Account.View.GetList(creds);
```
